redeploy.sh

```diff
@@ -18,15 +18,29 @@ cd /home/centos/hubmap/portal-ui/compose
 echo 'portal running?' `docker ps | grep portal-ui`
 
 echo 'stopping...'
-docker-compose -f hubmap.yml down
+
+# Also stop and remove the portal-ui-pord container on STAGE only
+if [ "$2" = "stage" ]; then
+    docker-compose -f hubmap.stage.yml down
+else
+    docker-compose -f hubmap.yml down
+fi
```

Comment:
I could be missing something, but I think you want `hubmap.stage.yml down` in addition to `hubmap.yml down` if appropriate, and it seems that only one or the other is being run... 
but I'm wondering if it'd be cleaner just to try it in every environment...
```suggestion
docker-compose -f hubmap.yml down

docker-compose -f hubmap.stage.yml down || echo 'No stage config: will continue...'
```